adam/curriculum/phase1_curriculum.py

```diff
@@ -113,25 +113,30 @@
     color_variable,
     object_variable,
     sampled,
+    TemplatePropertyVariable,
 )
 
 _CHOOSER = RandomChooser.for_seed(0)
 
-_Phase1InstanceGroup = InstanceGroup[  # pylint:disable=invalid-name
+_GROUND_OBJECT = object_variable("ground", GROUND)
+
+
+Phase1InstanceGroup = InstanceGroup[  # pylint:disable=invalid-name
     HighLevelSemanticsSituation,
     LinearizedDependencyTree,
     DevelopmentalPrimitivePerceptionFrame,
 ]
 
-_GROUND_OBJECT = object_variable("ground", GROUND)
-
 
-def _standard_object(
+def standard_object(
```

Comment:
Can you shift this and other code shared with the curriculum to a `curriculum_utils` module?